pageserver/src/tenant.rs

```diff
@@ -4020,7 +4070,7 @@ impl Tenant {
         new_timeline_id: TimelineId,
         idempotency: CreateTimelineIdempotency,
     ) -> Result<StartCreatingTimelineResult<'_>, CreateTimelineError> {
-        match self.create_timeline_create_guard(new_timeline_id) {
+        match self.create_timeline_create_guard(new_timeline_id, idempotency) {
```

Comment:
Why did you add the param here? It's passed on by the function. we could instead just use `idempotency` directly instead of `arg` in the match case below.